src/04-arrays.js

```diff
@@ -106,7 +121,10 @@ function removeElementFromEndOfArray(array) {}
  *  addElementToFrontOfArray([ 10 ], 9);
  *  //> [ 9, 10 ]
  */
-function addElementToFrontOfArray(array, element) {}
+function addElementToFrontOfArray(array, element) {
+  array.unshift(element);
```

Comment:
unshift does not take an argument. If you are ever unsure, make sure to take a look at documentation.